cwrapper.go

```diff
@@ -566,7 +569,7 @@ void ReleaseStringCritical(JNIEnv* env, jstring string, jchar* cstring) {
 	(*env)->ReleaseStringCritical (env, string, cstring);
 }
 
-jint AttachCurrentThread(JavaVM* vm, void** penv, void* args) {
+jint AttachCurrentThread(JavaVM* vm, JNIEnv** penv, void* args) {
```

Comment:
Is there a reason to use JNIEnv** ? In the OpenJDK jni.h its defined as void**

Reply:
Ok i see in the jni.h provided by android it is specified like this.

Reply:
Yeah, I changed this to fix build warnings. Didn't realise these were specced different across platforms. Not a huge issue.